examples/models/llama2/custom_ops/TARGETS

```diff
@@ -1,8 +1,22 @@
 # Any targets that should be shared between fbcode and xplat must be defined in
 # targets.bzl. This file can contain fbcode-only targets.
 
+load("@fbcode_macros//build_defs:python_unittest.bzl", "python_unittest")
```

Comment:
Use `runtime`? You can do

```
runtime.python_unittest(
...
)
```

Reply:
oh this was due to arc lint. let me fix.